chichiwang/exercise-js
# Move dir `pages/components/` to `app/components`

## Description
Linked to Issue: N/A
Move the entire contents of the dir `/pages/components` to `/app/components`. This makes more semantic/contextual sense - these are app-level components, although they are used by the pages. They are not, in themselves, pages.

## Changes
* Move the entire directory `/pages/components/` to `/app/components`
  * Update all references of `pages/components` to `app/components`

## Steps to QA
* Ensure all tests and linting passes
* Pull down and switch to this branch
* Run the application locally and ensure all elements on `/` and `/documentation` route renders correctly and without error
